tests/integration/test_exodus.py

```diff
@@ -9,6 +9,7 @@ def test_exodus_basic(cdn_test_url):
     )
 
     r = requests.get(url)
+    print(r.headers)
```

Comment:
How about formatting the json in an easier way to read it. 
https://www.journaldev.com/33302/python-pretty-print-json

Reply:
@lioramilbaum Yes! I will give it a try.

Reply:
@lioramilbaum done!